### PR TITLE
Create pql-.md

### PR DESCRIPTION
**Pull request by johndavidlugtu**
PQL definition in the handbook

### CODE REVIEW

File: operations/research-and-development/product/growth/pql-definition.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+### v1 PQL definition as of May 10, 2022
+
+A workspace / instance is considered a Product Qualified Lead (PQL) if it meets the criteria of any of the following categories:
```
Comment (jasonblais) [inline operations/research-and-development/product/growth/pql-definition.md]:
```suggestion
A workspace / instance is considered a Product Qualified Lead (PQL) if it is actively using Mattermost and meets one or more of the following criteria:
```

File: operations/research-and-development/product/growth/pql-definition.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+### v1 PQL definition as of May 10, 2022
```
Comment (jasonblais) [inline operations/research-and-development/product/growth/pql-definition.md]: Is it still the current definition?

**Review by jasonblais — CHANGES_REQUESTED** [review]
Thanks John! Left two comments. Thoughts on renaming the file to `pql.md` so that if we add more information about PQLs they can all be included on this page?

(It's fine if all it has right now is the definition)

File: operations/research-and-development/product/growth/pql-definition.md (hunk 2 above)
Comment (johndavidlugtu) [reply operations/research-and-development/product/growth/pql-definition.md]: Yes. Updating definition currently blocked by ability to view PQL -> opportunity data in SFDC. See [here](https://docs.google.com/document/d/1y_qSw7YEPtD_YQq19_BsV0eBl_Hngl-eQSeQu7WV2cE/edit#heading=h.mqhwsdw08d60) for details

**Comment by johndavidlugtu** [comment]
@jasonblais updated!

File: operations/research-and-development/product/growth/pql-definition.md (hunk 2 above)
Comment (jasonblais) [reply operations/research-and-development/product/growth/pql-definition.md]: @johndavidlugtu propose adding today's date, in that case?

**Comment by jasonblais** [comment]
@johndavidlugtu you'd need to rename the file itself to `pql.md`. Happy to show you if you get stuck
image

**Comment by johndavidlugtu** [comment]
Whoops -- how about now @jasonblais

**Review by jasonblais — APPROVED** [review]

**Comment by jasonblais** [comment]
Asking Justine for a quick editor pass, but looks good on my side, thanks John!

**Pull request merged**